### ISSUE TITLE
Tracking with continuous id crashes when does not find input boxes

### ISSUE THREAD

**Issue by photosartd** [IDEA-Research/Grounded-SAM-2#43]
In grounded_sam2_tracking_demo_with_continuous_id.py if `input_boxes.shape[0] == 0` at some point (which means there are no objects detected that correspond to the language prompt) the pipeline crashes at [this point in code](https://github.com/IDEA-Research/Grounded-SAM-2/blob/main/sam2/modeling/sam/mask_decoder.py#L203). Is it an unintended behaviour since if we didn't find any objects we just want to output the same frame?

**Comment by rentainhe** [comment]
Yes, if there're no object predicted, we should directly output the frame itself, we will update this code in the future release
